### ISSUE TITLE
[Feature request] Allow to pluralize collection property names

### ISSUE THREAD

**Issue by omatrot** [TrackableEntities/ObservableEntities.Core.Templates.TypeScript#1]
I may be wrong but I did not find an option to pluralize the collection property names in Many to One relationships.

**Comment by tonysneed** [comment]
Yes, I'm not pluralizing property names, but I am happy to accept a pull request at [EntityFrameworkCore.Scaffolding.Handlebars](https://github.com/TrackableEntities/EntityFrameworkCore.Scaffolding.Handlebars).

Otherwise, the pluralization I think can be handled by [Dcerialize](https://www.npmjs.com/package/dcerialize).
